phpmetrics.json

```diff
@@ -0,0 +1,164 @@
+{
+    "includes": [
+        "src"
+    ],
+    "exclude": [
+      "tests"
+    ],
+    "report": {
+        "html": "htmlout",
+        "csv": "/tmp/report.csv",
+        "json": "/tmp/report.json",
+        "violations": "/tmp/violations.xml"
+    },
+    "groups": [
+        {
```

Comment:
Please add the better-wp-package aswell

Reply:
BetterWPDB has been added, ScopableWP has been renamed and the deprecated ParameterBag has been removed